src/puppetlabs/puppetdb/cli/export.clj

```diff
@@ -24,20 +26,24 @@
             [slingshot.slingshot :refer [throw+ try+]])
   (:import [java.net URL]))
 
-(def ^:private api-version :v4)
+(def ^:private api-version :v1)
+(def ^:private api-command-versions
```

Comment:
it's a bit confusing that we're using "api" to refer to multiple APIs. "cli-api-version" and "command-versions" might be more specific.